### PR TITLE
feat: objects integration in firewall rules

### PR DESCRIPTION
**Pull request by andre8244**
https://github.com/NethServer/nethsecurity/issues/567

### CODE REVIEW

File: public/i18n/en/translation.json [hunk 1]
```diff
@@ -1845,12 +1855,12 @@
...
+      "name_helper": "Only letters and numbers allowed",
       "ip_version": "IP version",
-      "record": "Record | Records",
-      "host_record_tooltip": "Lorem ipsum dolor sit amet, consectetur adipisci elit, sed do eiusmod tempor incidunt ut labore et dolore magna aliqua.",
-      "host_set_records_tooltip": "Lorem ipsum dolor sit amet, consectetur adipisci elit, sed do eiusmod tempor incidunt ut labore et dolore magna aliqua.",
-      "domain_set_records_tooltip": "Lorem ipsum dolor sit amet, consectetur adipisci elit, sed do eiusmod tempor incidunt ut labore et dolore magna aliqua.",
+      "records": "Records",
+      "host_set_records_tooltip": "Every record is an item that composes the host set. Enter an IP address, a CIDR network (e.g. 10.10.10.0/24), an IP range (e.g. 10.10.10.1-10.10.10.5) or choose an object.",
+      "ip_family_host_set_tooltip": "Lorem ipsum dolor sit amet, consectetur adipisci elit, sed do eiusmod tempor incidunt ut labore et dolore magna aliqua.",
+      "ip_family_domain_set_tooltip": "Lorem ipsum dolor sit amet, consectetur adipisci elit, sed do eiusmod tempor incidunt ut labore et dolore magna aliqua.",
```
Comment (gsanchietti) [inline public/i18n/en/translation.json:1863]:
```suggestion
      "ip_family_domain_set_tooltip": "Entered domains will be resolved to IPv4 or IPv6 addresses.  Select the appropriate address family as needed by firewall rules.",
```

File: public/i18n/en/translation.json [hunk 2]
```diff
@@ -1845,12 +1855,12 @@
...
-      "name_helper": "Maximum 16 characters, no spaces",
+      "name_helper": "Only letters and numbers allowed",
       "ip_version": "IP version",
-      "record": "Record | Records",
-      "host_record_tooltip": "Lorem ipsum dolor sit amet, consectetur adipisci elit, sed do eiusmod tempor incidunt ut labore et dolore magna aliqua.",
-      "host_set_records_tooltip": "Lorem ipsum dolor sit amet, consectetur adipisci elit, sed do eiusmod tempor incidunt ut labore et dolore magna aliqua.",
-      "domain_set_records_tooltip": "Lorem ipsum dolor sit amet, consectetur adipisci elit, sed do eiusmod tempor incidunt ut labore et dolore magna aliqua.",
+      "records": "Records",
+      "host_set_records_tooltip": "Every record is an item that composes the host set. Enter an IP address, a CIDR network (e.g. 10.10.10.0/24), an IP range (e.g. 10.10.10.1-10.10.10.5) or choose an object.",
+      "ip_family_host_set_tooltip": "Lorem ipsum dolor sit amet, consectetur adipisci elit, sed do eiusmod tempor incidunt ut labore et dolore magna aliqua.",
```
Comment (gsanchietti) [inline public/i18n/en/translation.json:1862]:
```suggestion
      "ip_family_host_set_tooltip": "All items inside the host set must respected the selected family address.",
```

**Review by Tbaile — APPROVED** [review]

File: src/components/standalone/SideMenu.vue [hunk 3]
```diff
@@ -26,6 +26,7 @@ const loginStore = useLoginStore()
 const menuExpanded: any = ref({
   system: false,
   network: false,
+  'users-objects': false,
```
[the remark this answers is not in the document]
Comment (andre8244) [reply src/components/standalone/SideMenu.vue:29]: Everything breaks down :boom: It must match router path, and the usage of hyphens is recommended there

[the remark this answers is not in the document]
Comment (Tbaile) [reply src/components/standalone/SideMenu.vue:29]: Then don't mind me :laughing:

**Pull request merged**